### PR TITLE
Support Databox API version 3

### PR DESCRIPTION
**Pull request by VladaPetrovic**
Update SDK to support Databox API version 3

changes:
- [x] update `GET /lastpushes`
- [x] implement `GET /metrickeys` to return metric keys
- [x] implement `DELETE /data`
- [x] handle new response

### CODE REVIEW

File: src/main/java/com/databox/sdk/Databox.java [hunk 1]
```diff
@@ -78,7 +78,8 @@ private boolean push(String rawData) {
       conn = (HttpURLConnection) url.openConnection();
       conn.setRequestMethod("POST");
       conn.setRequestProperty("Content-Type", "application/json");
-      conn.setRequestProperty("User-Agent", "Databox/" + CLIENT_VERSION + " (Java)");
+      conn.setRequestProperty("User-Agent", "databox-java/" + CLIENT_VERSION);
+      conn.setRequestProperty("Accept", "application/vnd.databox.v" + string.split(CLIENT_VERSION)[0] + "+json");
```
Comment (umajeric) [inline src/main/java/com/databox/sdk/Databox.java]: `CLIENT_VERSION.split(".")[0]` namesto `string.split(CLIENT_VERSION)[0]`

File: src/main/java/com/databox/sdk/Databox.java [hunk 2]
```diff
@@ -128,7 +129,8 @@ private HttpURLConnection buildConnection(String method, String path) throws IOE
     String encodedToken = base64Encode((_token + ": ").getBytes("UTF-8"));
     connection.setRequestProperty("Authorization", "Basic " + encodedToken);
     connection.setRequestProperty("Content-Type", "application/json");
-    connection.setRequestProperty("User-Agent", "Databox/" + CLIENT_VERSION + " (Java)");
+    conn.setRequestProperty("User-Agent", "databox-java/" + CLIENT_VERSION);
+    conn.setRequestProperty("Accept", "application/vnd.databox.v" + string.split(CLIENT_VERSION)[0] + "+json");
```
Comment (umajeric) [inline src/main/java/com/databox/sdk/Databox.java]: `connection` namesto `conn`

Comment (umajeric) [inline src/main/java/com/databox/sdk/Databox.java]: `CLIENT_VERSION.split(".")[0]` namesto `string.split(CLIENT_VERSION)[0]`

File: src/main/java/com/databox/sdk/Databox.java [hunk 3]
```diff
@@ -128,7 +129,8 @@ private HttpURLConnection buildConnection(String method, String path) throws IOE
     String encodedToken = base64Encode((_token + ": ").getBytes("UTF-8"));
     connection.setRequestProperty("Authorization", "Basic " + encodedToken);
     connection.setRequestProperty("Content-Type", "application/json");
-    connection.setRequestProperty("User-Agent", "Databox/" + CLIENT_VERSION + " (Java)");
+    conn.setRequestProperty("User-Agent", "databox-java/" + CLIENT_VERSION);
```
Comment (umajeric) [inline src/main/java/com/databox/sdk/Databox.java]: `connection` namesto `conn`

**Comment by VladaPetrovic** [comment]
updated

**Comment by otobrglez** [comment]
Someone should write some tests for this amazing contributions. 🤔

**Pull request merged**